thewalrus/_montrealer.py

```diff
@@ -0,0 +1,135 @@
+"""
+Montrealer Python interface
+
+* Yanic Cardin and Nicolás Quesada. "Photon-number moments and cumulants of Gaussian states"
+  `arxiv:12212.06067 (2023) <https://arxiv.org/abs/arxiv:2212.06067v2>`_
+"""
+import numpy as np
+import numba
+from thewalrus.quantum.conversions import Xmat
+from thewalrus.charpoly import powertrace
+from ._hafnian import nb_ix
+from ._torontonian import tor_input_checks
+
+
+@numba.jit(nopython=True, cache=True)
+def dec2bin(num, n):  # pragma: no cover
+    """Helper function to generate convert an integer into an element of the powerset of n objects.
+
+    Args:
+        num (int): label to convert.
+        n (int): number of elements in the set.
+
+    Returns:
+        (array): array containing the labels of the elements to be selected.
+
+    """
```

Comment:
if descriptions are one-liners there is no need to add a period at the end
```suggestion
    """Helper function to convert an integer into an element of the power-set of ``n`` objects.

    Args:
        num (int): label to convert
        n (int): number of elements in the set

    Returns:
        (array): array containing the labels of the elements to be selected
    """
```